assets/src/domain/eventEditor/services/apollo/mutations/datetimes/useOptimisticResponse.ts

```diff
@@ -40,19 +40,21 @@ const useOptimisticResponse = (): OptimisticResCb => {
 		let espressoDatetime = {
 			__typename: 'EspressoDatetime',
 		};
+		// Get rid of null or undefined values
+		const filtereInput = filter(compose(not, isNil), input);
```

Comment:
In CRA I've notice `.filter(Boolean)` being used for the same purpose;
your solution is also nice

Reply:
`.filter(Boolean)` would filter out the values like `0`, `""`, `false`

Reply:
Here is the difference
![image](https://user-images.githubusercontent.com/18226415/74945869-72343e80-541e-11ea-8e38-18b4ce9f0267.png)